### PR TITLE
 Remove Global Pause and PausableToken

### PR DESCRIPTION
**Pull request by terryli0095**
Remove GlobalPause. And Pausable token. In the future pausing trueUSD will be done by upgrade the proxy to a paused version of trueUSD. Token controller's pauseTrueUSD method now calls upgradeTusdProxyImplTo to the paused version.
Reviewer @wjmelements

### CODE REVIEW

File: contracts/utilities/PausedTrueUSD.sol [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    function _decreaseApprovalAllArgs(address _spender, uint256 _subtractedValue, address _tokenHolder) internal {
+        revert("Token Paused");
+    }
+}
+
+/** @title DelegateERC20
+Accept forwarding delegation calls from the old TrueUSD (V1) contract. THis way the all the ERC20
+functions in the old contract still works (except Burn). 
+*/
+contract DelegateERC20 is PausedToken {
```
Comment (wjmelements) [inline contracts/utilities/PausedTrueUSD.sol]: Probably just inherit the original

Comment (terryli0095) [reply contracts/utilities/PausedTrueUSD.sol]: the original DelegateERC20 inherits ModularStandardToken it's cleaner this way

Comment (wjmelements) [reply contracts/utilities/PausedTrueUSD.sol]: Can rename it then

File: contracts/utilities/PausedTrueUSD.sol [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    event ChangeTokenName(string newName, string newSymbol);
+    event WipeBlacklistedAccount(address indexed account, uint256 balance);
+    event SetRegistry(address indexed registry);
+    event RedemptionAddress(address indexed addr);
+
+    function decimals() public returns (uint8) {
+        return DECIMALS;
+    }
+
+    function rounding() public returns (uint8) {
```
Comment (wjmelements) [inline contracts/utilities/PausedTrueUSD.sol]: view

Comment (wjmelements) [reply contracts/utilities/PausedTrueUSD.sol]: pure actually

File: contracts/utilities/UpgradeHelper.sol [hunk 3]
```diff
@@ -67,7 +67,7 @@ contract UpgradeHelper {
 
         // Configure TrueUSD
         tokenController.setTusdRegistry(registry);
-        tokenController.setGlobalPause(globalPause);
+        // tokenController.setGlobalPause(globalPause);
```
Comment (wjmelements) [inline contracts/utilities/UpgradeHelper.sol]: rm

File: truffle.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ module.exports = {
     coverage: {
       host: "localhost",
       network_id: "*",
-      port: 8525,
+      port: 8515,
```
Comment (wjmelements) [inline truffle.js]: revert

**Pull request merged**